pages/index.md

```diff
@@ -10,7 +10,7 @@ og_description: "Start your summer with a week learning about Jesus and technolo
 
 * Technical activity week for 12–15s
 * 6th - 13th August 2023
-* Chafyn Grove School, Salisbury, UK
```

Comment:
Is this under debate?

Reply:
Nope, just recommended not to be super obvious online apparently